htlcswitch/switch_test.go

```diff
@@ -4290,9 +4290,9 @@ func TestSwitchDustForwarding(t *testing.T) {
 	// We'll test that once the default threshold is exceeded on the
 	// Alice -> Bob channel, either side's calls to SendHTLC will fail.
 	//
-	// Alice will send 354 HTLC's of 700sats. Bob will also send 354 HTLC's
+	// Alice will send 51 HTLC's of 700sats. Bob will also send 51 HTLC's
```

Comment:
nit: constant value is 50, not 51. Or do we send +1?

Reply:
yeah we need to send 51, fixed below!